src/PaymentMethodTag.php

```diff
@@ -108,12 +108,17 @@ public function handler( $tag ) {
 
 		foreach ( $payment_methods as $payment_method ) {
 			$value = $payment_method->get_id();
-			$label = $payment_method->get_name();
 
 			if ( ! empty( $tag->values ) && ! \array_key_exists( $value, $pipes ) ) {
 				continue;
 			}
 
+			$label = $payment_method->get_name();
+
+			if ( ! empty( $tag->values ) && \array_key_exists( $value, $pipes ) ) {
```

Comment:
Is the `! empty( $tag->values )` condition needed here?

Reply:
Not needed, removed in https://github.com/pronamic/wp-pronamic-pay-contact-form-7/pull/26/commits/d293d81dee3a238ddbcb4c0dc0040f923a9730ec.